### PR TITLE
feat: Prateek | Test | Architecture-v2 to Test env.

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.html [hunk 1]
```diff
@@ -58,6 +58,14 @@ <h2>No Import Status Available.</h2>
                             <div class="col-xs-9 col-sm-9">{{ item?.submittedBy }}</div>
 
                         </div>
+                        <div class="row mt-1">
+
+                            <div class="col-xs-3 col-sm-3">
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/import-excel/import-report/import-report.component.html:63]: update **col-xs-3** to **col-3**

File: apps/web-giddh/src/app/import-excel/import-report/import-report.component.html [hunk 2]
```diff
@@ -58,6 +58,14 @@ <h2>No Import Status Available.</h2>
...
+                                <label class="text-gray">Branch:</label>
+                            </div>
+                            <div class="col-xs-9 col-sm-9">{{ item?.branch?.name || '-' }}</div>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/import-excel/import-report/import-report.component.html:66]: update **col-xs-9** to **col-9**

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/companyImportExport/component/form/company-import-export-form.html [hunk 3]
```diff
@@ -2,6 +2,33 @@
...
+                                        <a href="javascript:void(0)" class="list-item d-flex">
+                                            <span class="entity-icon">
+                                                <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch"
+                                                    tooltip="Branch"></i>
+                                                <i class="icon-company-HO pull-right"
+                                                    *ngIf="!option.parentBranch && !option.isCompany" tooltip="HO Branch"></i>
+                                            </span>
+                                            <span class="entity-details">
+                                                <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                                <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/companyImportExport/component/form/company-import-export-form.html:22]: Please change this class name to -  item-unique @prateek-giddh

File: apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.scss [hunk 4]
```diff
@@ -112,6 +112,30 @@
     border-radius: 4px;
     color: #cccccc;
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.scss:113]: Please give color var to this class @prateek-giddh

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts [hunk 5]
```diff
@@ -194,7 +208,49 @@ export class TbPlBsFilterComponent implements OnInit, OnDestroy {
                 this.filterData();
             }
         });
-
+        this.store.pipe(
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts:211]:
```suggestion
        this.store.pipe(select(state => state.company && state.company.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
            if(activeCompany) {
                this. activeCompany = activeCompany;
            }
        });
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Review by shalinee-walkover — APPROVED** [review]

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.html [hunk 6]
```diff
@@ -1,6 +1,32 @@
...
+                                <a href="javascript:void(0)" class="list-item d-flex">
+                                    <span class="entity-icon">
+                                        <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch"
+                                            tooltip="Branch"></i>
+                                        <i class="icon-company-HO pull-right"
+                                            *ngIf="!option.parentBranch && !option.isCompany" tooltip="HO Branch"></i>
+                                    </span>
+                                    <span class="entity-details">
+                                        <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                        <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/contact/aging-report/aging-report.component.html:20]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 7]
```diff
@@ -68,6 +68,32 @@
...
+                                            <a href="javascript:void(0)" class="list-item d-flex">
+                                                <span class="entity-icon">
+                                                    <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch"
+                                                        tooltip="Branch"></i>
+                                                    <i class="icon-company-HO pull-right"
+                                                        *ngIf="!option.parentBranch && !option.isCompany" tooltip="HO Branch"></i>
+                                                </span>
+                                                <span class="entity-details">
+                                                    <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                                    <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/contact/contact.component.html:87]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 8]
```diff
@@ -11,7 +11,32 @@
...
+                                <a href="javascript:void(0)" class="list-item d-flex">
+                                    <span class="entity-icon">
+                                        <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch"
+                                            tooltip="Branch"></i>
+                                        <i class="icon-company-HO pull-right"
+                                            *ngIf="!option.parentBranch && !option.isCompany" tooltip="HO Branch"></i>
+                                    </span>
+                                    <span class="entity-details">
+                                        <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                        <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/daybook/daybook.component.html:30]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.html [hunk 9]
```diff
@@ -1,6 +1,37 @@
...
+                        <ng-container>
+                            <a href="javascript:void(0)" class="list-item d-flex">
+                                <span class="entity-icon">
+                                    <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                    <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                        tooltip="HO Branch"></i>
+                                </span>
+                                <span class="entity-details">
+                                    <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                    <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.html:24]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.html [hunk 10]
```diff
@@ -2,6 +2,29 @@
...
+                            <ng-container>
+                                <a href="javascript:void(0)" class="list-item d-flex">
+                                    <span class="entity-icon">
+                                        <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                        <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                            tooltip="HO Branch"></i>
+                                    </span>
+                                    <span class="entity-details">
+                                        <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                        <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.html:19]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 11]
```diff
@@ -99,6 +99,32 @@
...
+                                <ng-container>
+                                    <a href="javascript:void(0)" class="list-item d-flex">
+                                        <span class="entity-icon">
+                                            <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                            <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                                tooltip="HO Branch"></i>
+                                        </span>
+                                        <span class="entity-details">
+                                            <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                            <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/ledger/ledger.component.html:118]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/manufacturing/report/mf.report.component.html [hunk 12]
```diff
@@ -1,14 +1,41 @@
...
+                                <ng-container>
+                                    <a href="javascript:void(0)" class="list-item d-flex">
+                                        <span class="entity-icon">
+                                            <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                            <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                                tooltip="HO Branch"></i>
+                                        </span>
+                                        <span class="entity-details">
+                                            <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                            <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/manufacturing/report/mf.report.component.html:29]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 13]
```diff
@@ -6,18 +6,40 @@
...
+                                <ng-container>
+                                    <a href="javascript:void(0)" class="list-item d-flex">
+                                        <span class="entity-icon">
+                                            <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                            <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                                tooltip="HO Branch"></i>
+                                        </span>
+                                        <span class="entity-details">
+                                            <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                            <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:23]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/reports/components/purchase-register-component/purchase.register.component.html [hunk 14]
```diff
@@ -10,14 +10,37 @@
...
+                    <ng-container>
+                        <a href="javascript:void(0)" class="list-item d-flex">
+                            <span class="entity-icon">
+                                <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                    tooltip="HO Branch"></i>
+                            </span>
+                            <span class="entity-details">
+                                <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/reports/components/purchase-register-component/purchase.register.component.html:27]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.html [hunk 15]
```diff
@@ -12,14 +12,37 @@
...
+                    <ng-container>
+                        <a href="javascript:void(0)" class="list-item d-flex">
+                            <span class="entity-icon">
+                                <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                    tooltip="HO Branch"></i>
+                            </span>
+                            <span class="entity-details">
+                                <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.html:29]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 16]
```diff
@@ -3,6 +3,29 @@
...
+                            <ng-container>
+                                <a href="javascript:void(0)" class="list-item d-flex">
+                                    <span class="entity-icon">
+                                        <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                        <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                            tooltip="HO Branch"></i>
+                                    </span>
+                                    <span class="entity-details">
+                                        <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                        <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:20]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/search/components/sidebar-components/search.sidebar.component.html [hunk 17]
```diff
@@ -2,6 +2,35 @@
...
+                                        <a href="javascript:void(0)" class="list-item d-flex">
+                                            <span class="entity-icon">
+                                                <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch"
+                                                    tooltip="Branch"></i>
+                                                <i class="icon-company-HO pull-right"
+                                                    *ngIf="!option.parentBranch && !option.isCompany" tooltip="HO Branch"></i>
+                                            </span>
+                                            <span class="entity-details">
+                                                <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                                <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/search/components/sidebar-components/search.sidebar.component.html:23]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.html [hunk 18]
```diff
@@ -7,9 +7,31 @@
...
+                                    <ng-container>
+                                        <a href="javascript:void(0)" class="list-item d-flex">
+                                            <span class="entity-icon">
+                                                <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                                <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                                    tooltip="HO Branch"></i>
+                                            </span>
+                                            <span class="entity-details">
+                                                <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                                <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.html:25]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/vat-report/vatReport.component.html [hunk 19]
```diff
@@ -6,6 +6,37 @@
...
+                                                            <a href="javascript:void(0)" class="list-item d-flex">
+                                                                <span class="entity-icon">
+                                                                    <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch"
+                                                                        tooltip="Branch"></i>
+                                                                    <i class="icon-company-HO pull-right"
+                                                                        *ngIf="!option.parentBranch && !option.isCompany" tooltip="HO Branch"></i>
+                                                                </span>
+                                                                <span class="entity-details">
+                                                                    <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                                                    <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/vat-report/vatReport.component.html:29]: change class name - item-unq to item-unique

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 20]
```diff
@@ -121,9 +147,33 @@
...
+                                    <ng-container>
+                                        <a href="javascript:void(0)" class="list-item d-flex">
+                                            <span class="entity-icon">
+                                                <i class="icon-branch-icon pull-right" *ngIf="option.parentBranch" tooltip="Branch"></i>
+                                                <i class="icon-company-HO pull-right" *ngIf="!option.parentBranch && !option.isCompany"
+                                                    tooltip="HO Branch"></i>
+                                            </span>
+                                            <span class="entity-details">
+                                                <div class="item giddh-limit single-line-clamp">{{ option.label || '-' }}</div>
+                                                <div class="item-unq giddh-limit single-line-clamp">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/ledger/ledger.component.html:165]: change class name - item-unq to item-unique

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**